package.json

```diff
@@ -1,28 +1,46 @@
 {
     "name": "@doist/reactist",
-    "version": "3.1.3",
     "description": "Open source React components by Doist",
-    "main": "./dist/reactist.js",
+    "author": "Henning Muszynski <henning@doist.com> (http://doist.com)",
+    "version": "4.0.0",
```

Comment:
btw: do you want to work against a v4 branch and do all changes there or release multiple (major) versions as you go?

Reply:
I want to make sure every release we do is compatible with Twist, so yeah, that might mean multiple releases. If we follow semver properly, I think this one should be a major one as we're thinking of changing the external API surface, but perhaps the next few need not be.

Before I cut a release though, I want to test a Twist PR that changes imports.